src/app/api/v1/jit.py

```
@@ -123,14 +123,28 @@ async def serve_hls_file(
     if not file_path or not file_path.exists():
         # 尝试从元数据获取转码信息并启动后台转码
         try:
+            # 先查找现有的窗口，而不是硬编码使用窗口 0
```

Ruff (0.13.1)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)